src/Compiler/Frontend/GLSL/GLSLExtensions.cpp

```diff
@@ -31,22 +31,22 @@ const std::map<std::string, int>& GetGLSLExtensionVersionMap()
 
         // ARB
         { E_GL_ARB_arrays_of_arrays,                        430 },
-        { E_GL_ARB_compute_shader,                          110 },
-        { E_GL_ARB_cull_distance,                           110 },
+        { E_GL_ARB_compute_shader,                          430 },
```

Comment:
These versions look good, but I think it would have been better if they are in a separate pull request.

Reply:
IDK how, I am pretty noob in all those git stuff... :)